awless/install.ps1

```diff
@@ -44,14 +44,14 @@ IF (!(Test-Path -Path "$pkg_src_cmd")) {
 
     # Settle unpacked archive into place
     echo "Install Location: $pkg_src_cmd"
-    New-Item "$pkg_src_bin" -ItemType Directory -Force
```

Comment:
this should not be moved, only the instances that are directly nested in an if statement should be moved.